diff --git a/luhn.test.js b/luhn.test.js
index 6ae9b7c..ebb21c0 100644
--- a/luhn.test.js
+++ b/luhn.test.js
@@ -1,4 +1,3 @@
-const assert = require('assert');
 const luhn = require('./luhn.js');
 
 const testCases = [
deleted file mode 100644
@@ -14,5 +13,7 @@ const testCases = [
 ];
 
 for(const {str, result} of testCases){
-    assert(luhn.check(str)==result, `Failed test:${JSON.stringify({str, result})}`);
+    test(JSON.stringify({str, result}), ()=>{
+        expect(luhn.check(str)).toBe(result);
+    })
 }
